definitions/divorce/json/CaseType.json

```diff
@@ -2,7 +2,7 @@
   {
     "LiveFrom": "01/01/2017",
     "ID": "DIVORCE",
-    "Name": "Divorce case - v113.19",
+    "Name": "Divorce case - v113.20",
```

Comment:
FYI you may have to up this depending on other PRs

Reply:
That's fine. I'll ask @vianvi to let me change this before merging